novu/api/integration.py

```diff
@@ -109,3 +109,21 @@ def limit(self, channel: Channel) -> IntegrationChannelUsageDto:
         return IntegrationChannelUsageDto.from_camel_case(
             self.handle_request("GET", f"{self._integration_url}/{channel}/limit")["data"]
         )
+
+
+
+    def setPrimary(self, integration_id: str) -> bool:
```

Comment:
All methods should be declared using "snake_case"

```suggestion
    def set_primary(self, integration_id: str) -> bool:
```